numpy/lib/polynomial.py

```diff
@@ -398,6 +398,11 @@ def polyfit(x, y, deg, rcond=None, full=False, w=None, cov=False):
     to points `(x, y)`. Returns a vector of coefficients `p` that minimises
     the squared error.
 
+    Please use `numpy.polynomial.Polynomial.fit` for new code. This remains
```

Comment:
Should we refer to `np.polynomial.polynomial.polyfit` instead? Or is the class method better?

Reply:
Never mind, I now saw @charris's comment that in fact the class method is recommended.

Reply:
The class method is numerically better, but to get the conventional coefficients one needs to do `p.convert()` on the resulting `p`.

Reply:
Could you suggest an alternative wording?

Reply:
Maybe,
"The `numpy.polynomial.Polynomial.fit` class method is recommended for new code as it is more stable numerically. See the documentation for the method for more information."

We might want to add a mention to Polynomial documentation about how to retrieve the conventional coefficients in the `Returns` section. The documentation is located in `numpy/polynomial/_polybase.py`.